Gemfile

```diff
@@ -1,2 +1,13 @@
 source "https://rubygems.org"
 gemspec
+
+# Extra optional dependencies so it works better out of the box regardless of the runtime environment
```

Comment:
Apparently it causes issues with the jekyll/builder to set the gem in the Gemfile 🤔 which is a no go for GitHub pages compatibility